owncloudApp/src/androidTest/java/com/owncloud/android/utils/IntentsUtil.kt

```diff
@@ -48,3 +48,13 @@ fun mockIntent(
     val intentResult = Instrumentation.ActivityResult(resultCode, result)
     intending(hasAction(action)).respondWith(intentResult)
 }
+
+@JvmName("mockIntentNoExtras")
+fun mockIntent(
+    resultCode: Int = Activity.RESULT_OK,
+    action: String
+) {
+    val result = Intent()
+    val intentResult = Instrumentation.ActivityResult(resultCode, result)
+    intending(hasAction(action)).respondWith(intentResult)
+}
```

Comment:
Add an empty line at the end of the file 👍

Reply:
Sorry I'll never learn!! 🥴

Reply:
You could run `ktLint` during pull request to check this. But because of your hidden CI, you have to do it by yourself

Reply:
Thanks for the suggestion @hannesa2. I like the idea of `ktLint`, `Detekt` and these types of tools. I have been considering adding them to this project, but I have not a lot of experience with them. That's why I have not included them yet. But I would like to.

Reply:
Adding it is not a big deal #3165 but you have to find your own rules